frontend/src/App.js

```diff
@@ -1,25 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import SignUp from "./pages/User/SignUp/SignUp";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+const App = () => {
+    return (
+        <div>
+            <SignUp />
+        </div>
+    );
```

Comment:
js files should have 2-space indentations.
Please update other files as well.

Then you can set vscode to make this as a default